src/ansys_sphinx_theme/search/fuse_search.py

```diff
@@ -29,84 +29,124 @@
 
 
 class SearchIndex:
-    """Class to get search index."""
+    """Generates a search index for a Sphinx document."""
```

Comment:
Verb must be imperative in docstrings:

```suggestion
    """Generate a search index for a Sphinx document."""
```